model_analyzer/config/run/model_run_config.py

```diff
@@ -96,7 +97,13 @@ def representation(self):
         Returns a representation string for the ModelRunConfig that can be used
         as a key to uniquely identify it
         """
-        return self.perf_config().representation()
+        repr_string = self.perf_config().representation()
+        repr_list = self._remove_mrc_from_list(repr_string.split())
+
+        return ' '.join(repr_list)
+
+    def _remove_mrc_from_list(self, repr_list: List[str]):
```

Comment:
what is the input here?
Is it something like this?
["-optionA", 10, "-optionB", 15]

or like this?
["-optionA=10", "-optionB=15"]

If it is the first one, this code won't work, because you are only removing the key but not the value (which is the real problem)

The test below implies this is working fine, but I wanted to double check

Reply:
The code seems to assume that there are no spaces between the option and the value, or inside of the option itself.  I doubt that would change, but it doesn't seem robust.  If the input was somehow something with a space like `--measurement-request-count="5, 10"` then this wouldn't work?

Reply:
It's both! Some options like batch size are '-b <number>', which will get split into two fields. Other's are like MRC, which have the format "measurement-request-count=<number>' all as one string.

There's no easy way to make this robust - you need knowledge of how it's going to be represented in the string from PA (which I admit is ugly).